deploy_remote.py

```diff
@@ -0,0 +1,159 @@
+#!/usr/bin/env python3
+# Copyright 2018 Iguazio
```

Comment:
```suggestion
# Copyright 2023 Iguazio
```

Reply:
wasn't sure it will pass the CI license test, as it specifically states 2018...

Reply:
It passes. It only has 2018 in the example.